_usage/mvu.md

```diff
@@ -0,0 +1,156 @@
+
+- Babelfish 2.2.0, built with PostgreSQL 14.5
+
+If you are not running version 1.3.1, you should first perform a minor version upgrade to version 1.3.1.  To perform a minor version upgrade:
+
+- Follow the [instructions](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_1_3_STABLE/contrib/README.md#build-the-postgres-engine) in the `babelfish_extensions` repository to build and replace the Babelfish PostgreSQL server with version 13.6 or higher.  
+
+- Build and replace your Babelfish extensions with extension version 1.2.2;  follow the [instructions](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_1_3_STABLE/contrib/README.md#install--build-dependencies) in the `babelfish_extensions` repository to build the extensions and copy them into the Babelfish binary directory. Finally, rename binary directory to *postgres_old* to prevent a naming conflict with the new binary directory.
+
+If you are running version 1.3.1, use the following steps to perform an MVU on a Babelfish server:
+
+1. Follow the [instructions at the Babelfish repo](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/README.md#build-the-postgres-engine) to build and install the Babelfish version 2.2.0 PostgreSQL server and extensions.
+
+2. Then, initialize the `data` directory for new engine version. For convenience, add /opt/mssql-tools/bin/ and ~/postgres/bin to your PATH environment variable. This enables you to run the tools without specifying the full path. For Z shell (the default shell for Cloud Desktop), add the following lines to your ~/.zshrc file:
+
+```sql
+export PATH=/opt/mssql-tools/bin:$PATH
+export PATH=~/postgres/bin:$PATH
+```
+
+3. Adjust the paths of the following commands to point to the directory of the target server, and run them to start the Postgres server:
+
+```sql
+~/postgres/bin/initdb -D ~/postgres/data/
+~/postgres/bin/pg_ctl -D ~/postgres/data/ -l logfile start
+```
+
+4. Modify the `~/postgres/data/postgresql.conf` file by uncommenting the following parameters and including the `babelfishpg_tds` extension in the `shared_preload_libraries` parameter:
+
+```sql
+listen_addresses = '*'
+shared_preload_libraries = 'babelfishpg_tds'
+```
+
+5. Modify `~/postgres/data/pg_hba.conf` to [allow connections](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html) from your IP address:
+
+```sql
+host    all             all     10.x.y.z/32            trust
+```
+
+6. Use the following command to apply the changes:
+
+```sql
+~/postgres/bin/pg_ctl -D ~/postgres/data/ -l logfile restart
+```
+
+Note that new server should not contain a Babelfish database, or have the extensions installed.
+
+7. Use `pg_ctl` to stop both `source` and `target` servers:
+
+  On the old server:
+
+```sql
+  ~/postgres_old/bin/pg_ctl -D ~/postgres_old/data/ stop
+```
+
+  On the new server:
+
+```sql  
+  ~/postgres_new/bin/pg_ctl -D ~/postgres_new/data/ stop
```

Comment:
Here we are introducing `postgres_new` directory but we have not mentioned it anywhere above. I think we should specify `postgres_new` directory above around line number 29.